index.html

```diff
@@ -152,10 +152,10 @@ <h1 class="project-title-tonic">Tonic</h1>
     
     
         <section class="works-card">
```

Comment:
- [ ] The project cards have alternating positions of the text and images. the first card has the image to the left and text to the right while the second card has an image to the right and the text to the left. Your project has all text to the left and the images to the right, kindly fix this.

Given design

Your design
![image](https://user-images.githubusercontent.com/81932478/179004130-f6730919-f013-47e7-9c4b-8e493851d765.png)

Reply:
- [ ] The see project button in the project cards should have a link to the live demo version of the project or to your github profile. Kindly add this as it is a requirement.
![image](https://user-images.githubusercontent.com/81932478/179005880-fecf03b6-7b48-4ba3-8788-d5b6b01afd95.png)